pyproject.toml

```diff
@@ -17,6 +17,7 @@ include = '\.pyi?$'
 [tool.pytest.ini_options]
 addopts = """
     -p no:ape_test
+    -p no:pytest_ethereum
```

Comment:
where is this coming from? saw it in a few PRs now, IIRC this was actually an old library I built that shouldn't be in use anymore, so interested where it's coming from

Reply:
i think it ships with web3.py on newer versions.

Reply:
oh maybe not, i cant find it.. i have no idea how or why this got installed but it only causes problems.
maybe i can uninstall.

Reply:
ya i confirmed it was my env that was messed up. reinstalling ape makes the problem go away..

however, pytest recommends disabling any unused plugins for performance reasons, so in the case this is installed, this is good anyway.